pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -222,6 +222,21 @@ func (cm *containerManagerImpl) GetNodeAllocatableReservation() v1.ResourceList
 	return result
 }
 
+// Returns error if the configuration is invalid, nil otherwise.
```

Comment:
Nit: Personally don't believe this comment offers any benefit.

Instead, maybe a high level description of why we use this given implementation for validating reserved resources?

Reply:
Agree with nit.

Maybe `verifyReservedResourcesHaveCapacity` ?

Reply:
Agree.